.github/workflows/ci-pipeline.yml

```diff
@@ -251,23 +252,6 @@ jobs:
         if: ${{ failure() }}
         run: find . -name config.log -exec cat {} \;
 
-  Build_gcc_u20:
```

Comment:
Could be changed to ubuntu 18 build. Test cases for GCC versions 7 and 8.

Reply:
Done in V2.